src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -138,42 +138,52 @@ public void postCommentPerIssue(String project, String repository, String pullRe
       
       // Severity available to create a task
       List<String> taskSeverities = getReportedSeverities();
-      
+
+      // Let's call this "issue_loop"
       for (SonarQubeIssue issue : issueReport.getIssues()) {
```

Comment:
False-positive that will marked as such after the merge (see: http://sonarqube-archive.15.x6.nabble.com/Sonar-4-4-rule-squid-S135-false-positive-Loops-should-not-contain-more-than-a-single-quot-break-quott-td5028139.html)